source/distributed/p4est_wrappers.cc

```diff
@@ -834,6 +834,73 @@ namespace internal
       return ((coarse_grid_cell >= parallel_forest->first_local_tree) &&
               (coarse_grid_cell <= parallel_forest->last_local_tree));
     }
+
+
+
+    template <>
+    bool
+    quadrant_is_equal<1>(const typename types<1>::quadrant &q1,
```

Comment:
This is the problem -- you're trying to specialize a template, but no template is available. Just *overload* the function name.

Reply:
Thanks @bangerth! I rather moved these classes and functions to `p4est_wrappers.h` and `p4est_wrappers.cc`. That worked!